markdown/tuto-020-shell-tools.md

```diff
@@ -278,4 +295,28 @@ to get a reminder.
 
 </div>
 
+<!-- ------- BALEINE ------------>
+<div id="BALEINE" class="tab-pane fade" markdown="1">
+
+## Baleine
+
+Baleine is a gateway tool allowing you to deploy Docker containers inside the R2Lab testbed.
+
+We will now demonstrate the deploy subcommand that will be very useful to you :
+
+```
+baleine deploy --image faraday.repo/tutorial --nodes 1 2 --options -t -d
+```
+
+The `deploy` subcommand allows you to pull and deploy the docker image selected with `--image` on the nodes selected with `--nodes`.
+The `--options` option allows you to give an array of argument to pass directly to the Docker Runtime.
+
+The `faraday.repo/tutorial` image is based on Ubuntu, so the `-t` and `-d` options are necessary for the Docker image to continue running.
```

Comment:
I sometimes feel like it would be helpful to stress either "hard-drive image" or "docker image", even when it sometimes feels obvious; but maybe that's just me...

also here, to continue after what ??

Reply:
Corrected.